### PR TITLE
Add keyevent for "home" key

### PR DESCRIPTION
**Pull request by stefanhoth**
This one was missing from the list of predefined script events.

### CODE REVIEW

File: gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Input.groovy [hunk 1]
```diff
@@ -62,6 +61,10 @@ class Input extends AdbTask {
     void clear() {
         key 28
     }
+    
+    void home() {
```
Comment (friedger) [inline gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Input.groovy]: Looks like random ordering of these events.

Comment (stefanhoth) [inline gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Input.groovy]: I put it at the end of the keys. Shall I reorder them instead?

Comment (friedger) [inline gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Input.groovy]: Yes, please.

Comment (stefanhoth) [inline gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Input.groovy]: @friedger done in e47c383

**Comment by friedger** [comment]
I like that :+1:

**Comment by devisnik** [comment]
:+1:

**Pull request merged**